Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp

```diff
@@ -48,16 +48,30 @@ namespace O3DE::ProjectManager
 
         m_repoPath = new FormLineEditWidget(tr("Remote URL"), "", this);
         m_repoPath->setMinimumSize(QSize(600, 0));
+        m_repoPath->lineEdit()->setPlaceholderText("http://github.com/example.git");
         vLayout->addWidget(m_repoPath);
 
         vLayout->addSpacing(10);
 
+        QHBoxLayout* warningHLayout = new QHBoxLayout(this);
+
+        QLabel* warningIcon = new QLabel(this);
```

Comment:
```suggestion
        QHBoxLayout* warningHLayout = new QHBoxLayout();
        QLabel* warningIcon = new QLabel();
```
These should get automatically parented when they are added to layouts